bridges/SteamBridge.php

```diff
@@ -8,45 +8,10 @@ class SteamBridge extends BridgeAbstract {
 	const MAINTAINER = 'jacknumber';
 	const PARAMETERS = array(
 		'Wishlist' => array(
-			'username' => array(
-				'name' => 'Username',
+			'userid' => array(
+				'name' => 'Steamid64 (find it on steamid.io)',
 				'required' => true,
```

Comment:
It probably makes sense to hint the format of a steam ID (you can also have a regex to check the format).

```suggestion
				'required' => true,
				'exampleValue' => '76561198821231205',
				'pattern' => '[0-9]{17}',
```